src/validation.cpp

```diff
@@ -502,7 +505,7 @@ class MemPoolAccept
                             /* m_bypass_limits */ false,
                             /* m_coins_to_uncache */ coins_to_uncache,
                             /* m_test_accept */ false,
-                            /* m_allow_bip125_replacement */ false,
+                            /* m_allow_bip125_replacement */ true,
```

Comment:
This RBF policy is essentially treating the package as "one big transaction," which works because everything is in the ancestor set of the child. PackageTestAccept doesn't impose any topology restrictions - it even allows unrelated transactions to be submitted as a group. So it wouldn't make sense to use each other's fees.